services/elastic_ip_manager/src/main/java/com/futurewei/alcor/elasticipmanager/entity/ElasticIpAllocatedIpv6.java

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.elasticipmanager.entity;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+public class ElasticIpAllocatedIpv6 {
+
+    @JsonProperty("range_id")
+    private String rangeId;
+
+    @JsonProperty("allocated_ipv6")
+    private String allocatedIpv6;

```

Comment:
ElasticIpAllocatedIpv4 support a set of ip address while Ipv6 has an individual ip. Do we expect to extend it to a set later?